perceval/backends/git.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, uri, gitpath, cache=None, origin=None):
         self.gitpath = gitpath
 
     @metadata
-    def fetch(self, from_date=DEFAULT_DATETIME):
+    def fetch(self, from_date=DEFAULT_DATETIME, branches=[]):
```

Comment:
By the way, `GitCommand` should also support this feature. You need to manage it on the class. You can do the same I did on `telegram` class with the `chats` filter.

https://github.com/grimoirelab/perceval/blob/master/perceval/backends/telegram.py#L307